controls/radtreeview/how-to/wpf-windowsexplorer-like-treeview.md

```diff
@@ -319,54 +322,60 @@ This tutorial will combine in itself the following:
 
 	For better convenience, the __ServiceFacade__ class is an implementation of the [Singleton](http://en.wikipedia.org/wiki/Singleton_pattern) pattern. When the only one instance of the class is created, the __Drives__ collection needs to be populated with all drives on your machine. This will happen in the __Initialize()__ method. Here is a sample code how this can be achieved:

```

Comment:
When only one (without the).
**Example 6** **demonstrates** how this can be achieved.